local-remote-execution/README.md

```diff
@@ -1,45 +1,65 @@
 # Local Remote Execution
 
```

Comment:
I don't know if it should say "Trace Machina" in this document specifically, but we should be sure to say our name with some frequency in documentation generally for branding.

Reply:
This repo is less about Trace Machina and more about the NativeLink project.